src/engraving/libmscore/score.cpp

```diff
@@ -708,7 +708,8 @@ void Score::rebuildTempoAndTimeSigMaps(Measure* measure)
                 }
             }
             if (stretch != 0.0 && stretch != 1.0) {
-                BeatsPerSecond otempo = tempomap()->tempo(segment.tick().ticks());
+                BeatsPerSecond multiplier = tempomap()->tempoMultiplier();
```

Comment:
Why an extra variable it if is only used once?

Reply:
I chose this way because of line length and the fact that e.g. `ntempo` is used once also. But I understand that the name `ntempo` conveys some additional semantics, mine none, and line length would be not so excessive. Should I commit a single line assignment?

Reply:
What does the staff team think?